src/main/java/cubicchunks/CubicChunks.java

```diff
@@ -155,19 +155,49 @@ public static void addConfigChangeListener(IConfigUpdateListener listener) {
     }
 
     public static class Config {
+        
+        public static enum Options {
+            maxGeneratedCubesPerTick(1, Integer.MAX_VALUE, 49 * 16, "The number of cubic chunks to generate per tick."),
+            lightingTickBudget(1, Integer.MAX_VALUE, 10,
+                    "The maximum amount of time in milliseconds per tick to spend performing lighting calculations."),
+            verticalCubeLoadDistance(2, 32, 8, "Similar to Minecraft's view distance, only for vertical chunks."),
+            worldHeightLowerBound(AddressTools.MIN_BLOCK_Y, 0, -4096,
+                    "The lower boundary on the world. Blocks will not generate or load below this point."),
+            worldHeightUpperBound(256, AddressTools.MAX_BLOCK_Y, 4096,
+                    "The upper boundary on the world. Blocks will not generate or load above this point."),
+            chunkGCInterval(1, Integer.MAX_VALUE, 20 * 10,
```

Comment:
lowercase enums? really? Took me a while to figure out how you are calling a method without being in some method already...